spec/util/test_env.lua

```diff
@@ -429,7 +431,9 @@ local function hash_environment(path)
       return execute_output("find " .. path .. " -type f -exec stat -f \"%z %N\" {} \\; | md5")
    elseif test_env.TEST_TARGET_OS == "windows" then
       return execute_output("\"" .. Q(test_env.testing_paths.win_tools .. "/find") .. " " .. Q(path)
-         .. " -printf \"%s %p\"\" > temp_sum.txt && certUtil -hashfile temp_sum.txt && del temp_sum.txt")
+         .. " -printf \"%s %p\"\" > temp_sum.txt && "
+         .. "C:\\Windows\\System32\\certUtil -hashfile temp_sum.txt && "
```

Comment:
This ensure using Windows' `certutil`. If we use `certutil` here, `certutil` provided by MSYS2 is used.

Reply:
Sorry, I couldn't guarantee it but it works on all platforms that we use for our CI.
Generally, Windows keeps backward compatibility very well. So this will not be changed.